products/graalvm.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GraalVM
+category: lang
+iconSlug: NA
+permalink: /graalvm
+releasePolicyLink: https://www.graalvm.org/release-notes/release-calendar/
+changelogTemplate: "https://www.graalvm.org/release-notes/{{ '__RELEASE_CYCLE__' | replace:'.','_' }}/"
+LTSLabel: "<abbr title='Annual Release'>AR</abbr>"
+eolColumn: Maintenance
+releaseColumn: true
+releaseDateColumn: true
+
+# Disabled because tags date on https://github.com/graalvm/graalvm-ce-builds/tags are not the
```

Comment:
How many/Which of these are mismatching?

Reply:
Almost all of them [on the first tags page](https://github.com/graalvm/graalvm-ce-builds/tags). Most only by a few days, but some are really wrong.

For example for GraalVM 22.3.0 (the last version), vm-22.3.0 tag date is 2022-07-26, but the release date is 2022-10-25. 
If you go to this version [GitHub release page](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-22.3.0) the release date is OK, but you would have to use [the GitHub release API](https://github.com/endoflife-date/release-data/issues/34) to get it.

Reply:
Seeing as there are only gonna be quarterly updates here, manual updates should be good for now.

> It seems tags have been created a long time after the releases.

I was confused by this, which seems incorrect - the tags were created a long time _before the release_.

I did look at the [Docker Images](https://github.com/graalvm/container/pkgs/container/graalvm-ce/versions?filters%5Bversion_type%5D=tagged), but they are unreliable as well, as they seem to be overwritten.

Reply:
Ok, and sorry for the wrong commit message. I fixed it.